homeassistant/components/tuya/climate.py

```diff
@@ -430,10 +430,9 @@ def fan_mode(self) -> str | None:
     @property
     def fan_modes(self) -> list[str]:
         """Return fan modes for select."""
-        data = json.loads(
+        return json.loads(
             self.tuya_device.function.get(DPCODE_FAN_SPEED_ENUM, {}).values
```

Comment:
Side note: I missed this in the previous review. `json.loads` will error if we pass `dict.values` to it. We fixed this in some other places in the module. We need to check the `tuya_device.function.get` return value is truthy before passing it to json.loads.

Reply:
Pushed a change to add that check